addons/web/controllers/binary.py

```diff
@@ -89,6 +89,7 @@ def content_common(self, xmlid=None, model='ir.attachment', id=None, field='raw'
         '/web/assets/<int:id>-<string:unique>/<path:extra>/<string:filename>'], type='http', auth="public")
     # pylint: disable=redefined-builtin,invalid-name
     def content_assets(self, id=None, filename=None, unique=False, extra=None, nocache=False):
+        # See also base/models/ir_actions_report.py _get_web_assets_attachment
```

Comment:
Antony said we shouldn't loose time making stuff too much generic and that it was fine to duplicate code